common/service/unit.go

```diff
@@ -0,0 +1,87 @@
+// Handle changes to unit entities
+
+package service
+
+import (
+	"time"
+
+	"github.com/juju/juju/state"
+	"github.com/juju/juju/status"
+)
+
+// Handle a changed unit
+func (s *FakeJujuService) handleUnitChanged(id string) error {
+	log.Infof("Handling changed unit %s", id)
+
+	// Get the unit
+	unit, err := s.state.Unit(id)
+	if err != nil {
+		return err
+	}
+
+	agentStatus, err := unit.AgentStatus()
+	if err != nil {
+		return err
+	}
+
+	log.Infof("Agent Status %v %v", agentStatus.Status, status.Allocating)
```

Comment:
hmm, why are we logging `status.Allocating`? Looks like it might have been debugging code that's sneaked in?